ansible/roles/test/files/ptftests/mtu_test.py

```diff
@@ -143,11 +192,55 @@ def check_ip_mtu(self):
 
     #---------------------------------------------------------------------
 
+    def check_ipv6_mtu(self):
+        '''
+        @summary: Check unicast IPv6 forwarding in DUT works for MAX MTU.
```

Comment:
Same above.
The test logic looks good to me. Could you try to parametrize current test function and reuse the code?

Reply:
Fixed like above